### DISCUSSION TITLE
Add a delete button to the collection detail page as well

### DISCUSSION THREAD

**Discussion by nirmalburlakoti** [directus/directus#12077] [Feature Requests]
I have been testing Directus using free community cloud service. I added a new data model and enabled all optional fields. I then continued to add new fields but I could not. I got [UNKNOWN] Network Error. I tried deleting the data model from the list of all data models but received the same error. Then I realized the delete button on the top right while editing the data model was completely missing.
![image](https://user-images.githubusercontent.com/42568396/157612577-5c2ab60b-c662-4463-abda-c88815e60e29.png)
![image](https://user-images.githubusercontent.com/42568396/157612666-a83c76c4-a99a-47bc-a94f-1a73c516f68e.png)
The complete error I received upon clicking on the copy icon is: 

{
  "stack": "Error: Network Error\n    at TT (https://lucalhfg.directus.app/admin/assets/vendor.f59145b7.js:69:166494)\n    at XMLHttpRequest.v.onerror (https://lucalhfg.directus.app/admin/assets/vendor.f59145b7.js:70:2150)",
  "message": "Network Error",
  "config": {}
}

### To Reproduce

Just adding a data model caused this error.

### What version of Directus are you using?

9.5.2

### What version of Node.js are you using?

16.14.0

### What database are you using?

No Idea. Been using community cloud service.

### What browser are you using?

Microsoft Edge

### What operating system are you using?

macOS 12

### How are you deploying Directus?

Directus Cloud

**Comment by rijkvanzanten** [comment]
Heya! The delete action can be found under the menu on the overview page:

<img width="1056" alt="CleanShot 2022-03-10 at 09 50 35@2x" src="https://user-images.githubusercontent.com/9141017/157687230-815dcb33-b663-45d3-acfa-46c956ebe83d.png">

That being said, I agree you would expect it to be on the detail page as well. @azrikahar is that something you can add in?

**Reply by azrikahar** [reply]
Makes sense to me as well. Will do 👍

**Reply by azrikahar** [reply]
Wait.. actually this already exists! 😄

![chrome_s7GMM2owQK](https://user-images.githubusercontent.com/42867097/157845609-522b886c-c485-4c03-9573-d51776aca30c.png)

I believe the issue encountered by OP is because the delete button will only show up if the `item` exists:

https://github.com/directus/directus/blob/c1d705f94ad2061e6912d593a7183cb577cac6b2/app/src/modules/settings/routes/data-model/fields/fields.vue#L16

However as OP faced network error, the endpoint here fails so `item` doesn't get set:

https://github.com/directus/directus/blob/c1d705f94ad2061e6912d593a7183cb577cac6b2/app/src/composables/use-item/use-item.ts#L100

When I tried to manually cause it to fail by turning it into a malformed endpoint, I was able to get the same "no delete button" result:

![chrome_fpNwz9BOEZ](https://user-images.githubusercontent.com/42867097/157845979-65c88baa-4896-4733-a774-b0051a9ecfcd.png)

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
